src/python3/request/item_search.py

```diff
@@ -62,6 +44,19 @@ def get_async(self):
                                                     self.get)
         collection_response = yield from future
         return collection_response
+    
+    @staticmethod
+    def get_next_page_request(collection_page, client, options, q=None):
+        """Gets the ItemSearchRequest for the next page. Returns None if there is no next page
+
```

Comment:
Missed this before. Shouldn't these include a description of the arguments? This is especially important because it needs to be clear that `collection_page` needs to be the same type as the current class.